package.json

```diff
@@ -33,9 +33,10 @@
     "audit": "lerna run audit --stream && npm audit --production",
     "bootstrap": "lerna bootstrap",
     "bump": "lerna publish",
+    "commit": "cz",
```

Comment:
this is for `npm run commit`, which brings up an interactive CLI wizard to compose a commit message